### ISSUE TITLE
Can't get with apt

### ISSUE THREAD

**Issue by void-robotics** [ros/urdf_sim_tutorial#4]
Hello,

In trying to continue the [URDF tutorial](https://wiki.ros.org/urdf/Tutorials/Using%20a%20URDF%20in%20Gazebo) I get

```
nathan@nathan-vm:/opt/ros/kinetic/share/urdf_tutorial$ sudo apt-get install ros-kinetic-urdf-sim-tutorial
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libclang1-3.6 libllvm3.6v5 libllvm4.0:i386 libllvm5.0 libllvm5.0:i386 libqbscore1 libqbsqtprofilesetup1 libvtk-java libvtk5.10
  qml-module-qtquick-controls qtcreator-doc qtxmlpatterns5-dev-tools snap-confine snapd-login-service
Use 'sudo apt autoremove' to remove them.
The following NEW packages will be installed:
  ros-kinetic-urdf-sim-tutorial
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 8,168 B of archives.
After this operation, 84.0 kB of additional disk space will be used.
Err:1 http://packages.ros.org/ros/ubuntu xenial/main amd64 ros-kinetic-urdf-sim-tutorial amd64 0.3.0-1xenial-20190320-182239-0800
  404  Not Found [IP: 2600:3404:200:237::2 80]
E: Failed to fetch http://packages.ros.org/ros/ubuntu/pool/main/r/ros-kinetic-urdf-sim-tutorial/ros-kinetic-urdf-sim-tutorial_0.3.0-1xenial-20190320-182239-0800_amd64.deb  404  Not Found [IP: 2600:3404:200:237::2 80]

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
nathan@nathan-vm:/opt/ros/kinetic/share/urdf_tutorial$ 
```

Installing other ROS packages seem to work on my machine.

Thanks,
Nathan

**Comment by DLu** [comment]
I believe you need to run `sudo apt-get update` first to get the latest versions. It looks like you are trying to download an outdated version (`20190320`) when there is a more up to date version available ([`20190608`](http://packages.ros.org/ros/ubuntu/pool/main/r/ros-kinetic-urdf-sim-tutorial/ros-kinetic-urdf-sim-tutorial_0.3.0-1xenial-20190608-141925-0800_amd64.deb))

Please close this ticket if that solves your problem.

**Comment by void-robotics** [comment]
thanks; the key must've changed
